pages/index.js

```diff
@@ -195,7 +228,17 @@ class Index extends React.Component {
               </CardBody>
             </Card>
           )}
-          <Row className="equal-height mb-5">{queues}</Row>
+          <Row className="equal-height mb-5">
+            {queues}
+            <QueueEdit
+              /* eslint-disable react/prop-types */
```

Comment:
Why is the linter disabled here?

Reply:
This is one of the part that I'm not sure. So in the props of `QuestionList`, the `questions` is an id indexed object, so I can simply find the corresponding `Question` by id. But things are separated out for `queues` (and also `courses`, IIRC) in its container, `queues` is just a plain array that stores only all `Queue` objects while `queuesByID` is created in addition as the id indexed object for `Queue`. The old code seems never explicitly uses `queuesByID` (correct me if I'm wrong), so there was no eslint error for "reat/prop-types" rule even though we do not add `propTypes` check for `queuesByID`. But now since I need to retrieve a specific `Queue` by its id, it becomes a problem since I explicitly look up in `queuesByID`. Should I add `PropTypes` check for it then?

Reply:
Yep! That'd be great.

Reply:
Sure, will working on it later today.